tests/rbac_test.go

```diff
@@ -0,0 +1,120 @@
+package tests_test
+
+import (
+	"flag"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/ginkgo/extensions/table"
+	//"github.com/google/goexpect"
+	//"kubevirt.io/kubevirt/pkg/api/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"kubevirt.io/kubevirt/pkg/kubecli"
+	"kubevirt.io/kubevirt/tests"
+	ktests "kubevirt.io/kubevirt-ansible/tests"
+)
+
+var _ = Describe("Subresource Api", func() {
+
+	flag.Parse()
+
+	var vmi ktests.VMManifest
+	//var role string
+
+	virtClient, err := kubecli.GetKubevirtClient()
+	tests.PanicOnError(err)
+
+	BeforeEach(func() {
+		vmi.Name = "vm-cirros"
+		vmi.Manifest = "tests/manifests/vm-cirros.yaml"
+
+		By("Login with the user test_user")
+		args := []string{"login", "-u", "test_admin", "-p", "123456"  }
+		_, err := tests.RunCommand("oc", args...)
+		Expect(err).ToNot(HaveOccurred())
+
+		ktests.CreateUser()
+		ktests.CreateNamespaces()
+	})
+
+	AfterEach(func ()  {
+		By("Login with the user test_admin")
+		args := []string{"login", "-u", "test_admin", "-p", "123456"  }
+		_, err := tests.RunCommand("oc", args...)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Delete identity of user kubevirt-test-user")
+		args = []string{"delete", "identity", "allow_all_auth:kubevirt-test-user" }
+		_, err = tests.RunCommand("oc", args...)
+		Expect(err).ToNot(HaveOccurred())
+
+		ktests.DeleteUser()
+		ktests.RemoveNamespaces()
+	})
+
+	DescribeTable("Rbac Authorization", func (role string)  {
+
+			By("Add admin role to user kubevirt-test-user")
```

Comment:
looks like you're adding either the admin or edit role to the user.

Reply:
@davidvossel I'm adding both admin and edit role independently from DescribeTable Entry. I need to update the text like By("Add a role to user kubevirt-test-user ")

Reply:
how about use it like `By(fmt.Sprintf("Add %s role to user kubevirt-test-user", role))`